### ISSUE TITLE
Add gif support and gif picker

### ISSUE THREAD

**Issue by amywastaken** [revoltchat/desktop#35]
Inspired by Discord, an option should be added

**Comment by penloy** [comment]
Yeah I support a gif chooser option too. Something like Discord has with Tenor.

**Comment by CookieSource** [comment]
A gif chooser would be really nice

**Comment by Mar0xy** [comment]
This should be in [revite](https://github.com/revoltchat/revite) as that is the web interface which the desktop app also uses.
